### DISCUSSION TITLE
Vercel cron jobs not running on schedule

### DISCUSSION THREAD

**Discussion by wikskold** [vercel/community#2339] [Help]
### Summary

I can't get vercel cron jobs to run it's schedule but it works fine to run it manually at the press of the run button on the Cron Jobs page and in the logs I get back status code 200 and the code runs successfully.

My vercel.json file:

```
{
    "crons": [
        {
            "path": "/api/updateJobs",
            "schedule": "0 7 * * *"
        }
    ]
}
```

And the updateJobs file:

```
import { fetchJobs } from './fetchJobs';

export default async function handler(request, response) {
    await fetchJobs();
    response.status(200).send({ success: true });
}
```

This works only when manually triggered. (I have a hobby plan on vercel)
I run the project on node.js 16.x
Am I missing something?

**Comment by amyegan** [comment]
Hi @wikskold. Is this problem happening on a production deployment, or on a preview?

**Reply by wikskold** [reply]
Hi @amyegan. It's happening on a production deployment. I'm now running a cron job on github actions without any issues.
